internal/controller/fluxinstance_controller_test.go

```diff
@@ -219,6 +229,7 @@ func TestFluxInstanceReconciler_LifeCycle(t *testing.T) {
 
 func TestFluxInstanceReconciler_FetchFail(t *testing.T) {
 	g := NewWithT(t)
+	t.Setenv("NOTIFICATIONS_DISABLED", "yes")
```

Comment:
Since we are disabling notifications in most places in the tests I'd prefer disabling them by default inside the `getFluxInstanceReconciler()` function and enabling only in the tests that need it (if any).

Reply:
Done! Thanks